src/ui/map.js

```diff
@@ -818,6 +820,8 @@ class Map extends Camera {
      * Returns the map's minimum allowable pitch.
      *
      * @returns {number} minPitch
+     * @example
+     * const minPitch = map.getMinPitch();
```

Comment:
What's your opinion on adding examples that are very trivial like getters here?

I skipped some of these in the other PR (for things such as camera.getPosition() etc..), as it doesn't add a lot of help compared to reading the description of the API. But for consistency and completeness I'm not against being thorough so we can promote that to an error instead of a warning.

Reply:
From a docs team perspective, we get so much feedback from users asking for examples that we'd like to err on the side of consistency and completeness even when it means adding examples for fairly trivial things like this.